File: src/main/java/models/Tutor.java
package models;

public class Tutor {
    private int id;
    private String firstName;
    private String lastName;
    private TutorPreferences tutorPreferences;
    private TutorStatus tutorStatus;


    public Tutor(int id, String firstName, String lastName, TutorPreferences tutorPreferences, TutorStatus tutorStatus) {
        this.id = id;
        this.firstName = firstName;
        this.lastName = lastName;
        this.tutorPreferences = tutorPreferences;
        this.tutorStatus = tutorStatus;
    }

}

File: CourseRequirementGenerator/CourseRequirementGenerator/Tutor.cs
using System.Collections.Generic;

namespace CourseRequirementGenerator
{
    public class Tutor
    {
        public int id { get; set; }
        public string firstName { get; set; }
        public string lastName { get; set; }
        public string role { get; set; }
        public List<int> prefCourses { get; set; }
        public List<int> willingCourses { get; set; }
        public List<int> prefShift { get; set; }
        public List<int> willingShift { get; set; }
        public int? prefShiftAmount { get; set; }
        public int? willingShiftAmount { get; set; }
    }
}
